packages/gitgraph-node/src/index.ts

```diff
@@ -0,0 +1,21 @@
+import { GitgraphCore } from "gitgraph-core/lib";
+
+// Domain (rendering logic)
+import computeGraphMap from "./compute-graph-map";
+
+// Infrastructure (logger implementations)
+import consoleGraphLogger from "./console-graph-logger";
+
+function createGitgraph(): GitgraphCore {
+  // Limiting Gitgraph Core config options is intentional since most of them
+  // are not yet implemented in node.js.
+  // Templates options should be limited & customized for the console.
+  return new GitgraphCore();
```

Comment:
Fair, it went a bit too functional here. +1 for consistency.

Reply:
So I wonder: do we want to keep `render(gitgraph: Gitgraph)` or should it be `gitgraph.render()` since we use classes here?

I'd say the second one feels more intuitive using classes. But I do also like the render function. What do you think @fabien0102 ?